src/services/completions.ts

```diff
@@ -1060,22 +1110,30 @@ namespace ts.Completions {
                     symbols.push(firstAccessibleSymbol);
                     const moduleSymbol = firstAccessibleSymbol.parent;
                     symbolToOriginInfoMap[getSymbolId(firstAccessibleSymbol)] =
-                        !moduleSymbol || !isExternalModuleSymbol(moduleSymbol) ? { kind: SymbolOriginInfoKind.SymbolMemberNoExport } : { kind: SymbolOriginInfoKind.SymbolMemberExport, moduleSymbol, isDefaultExport: false };
+                        !moduleSymbol || !isExternalModuleSymbol(moduleSymbol)
+                            ? { kind: getNullableSymbolOriginInfoKind(SymbolOriginInfoKind.SymbolMemberNoExport) }
+                            : { kind: getNullableSymbolOriginInfoKind(SymbolOriginInfoKind.SymbolMemberExport), moduleSymbol, isDefaultExport: false };
                 }
                 else if (preferences.includeCompletionsWithInsertText) {
-                    addPromiseSymbolOriginInfo(symbol);
+                    addSymbolOriginInfo(symbol);
                     symbols.push(symbol);
                 }
             }
             else {
-                addPromiseSymbolOriginInfo(symbol);
+                addSymbolOriginInfo(symbol);
                 symbols.push(symbol);
             }
 
-            function addPromiseSymbolOriginInfo (symbol: Symbol) {
+            function addSymbolOriginInfo(symbol: Symbol) {
                 if (insertAwait && preferences.includeCompletionsWithInsertText && !symbolToOriginInfoMap[getSymbolId(symbol)]) {
-                    symbolToOriginInfoMap[getSymbolId(symbol)] = { kind: SymbolOriginInfoKind.Promise };
+                    symbolToOriginInfoMap[getSymbolId(symbol)] = { kind: getNullableSymbolOriginInfoKind(SymbolOriginInfoKind.Promise) };
                 }
+                else if (insertQuestionDot) {

```

Comment:
Only when `preferences.includeCompletionsWithInsertText` is true you can set insert text